t/sanity.t

```diff
@@ -468,3 +468,76 @@ lock 2: unlock: nil, unlocked
 --- no_error_log
 [error]
 
+
+
+=== TEST 14: serial lock and unlock with safe_add
+--- http_config eval: $::HttpConfig
+--- config
+    location = /t {
+        content_by_lua '
+            local lock = require "resty.lock"
+            for i = 1, 2 do
+                local lock = lock:new("cache_locks", {safe_add = true})
+                local elapsed, err = lock:lock("foo")
+                ngx.say("lock: ", elapsed, ", ", err)
+                local ok, err = lock:unlock()
+                if not ok then
+                    ngx.say("failed to unlock: ", err)
+                end
+                ngx.say("unlock: ", ok)
+            end
+        ';
+    }
+--- request
+GET /t
+--- response_body
+lock: 0, nil
+unlock: 1
+lock: 0, nil
+unlock: 1
+
+--- no_error_log
+[error]
+
+
+
+=== TEST 15: use safe_add, dont have enough memory
+--- http_config eval: $::HttpConfig
+--- config
+    location = /t {
+        content_by_lua '
+            local lock  = require "resty.lock"
+            local cache = ngx.shared.cache_locks
+
+            cache:flush_all()
+            function full_dict( dict )
+                for i=1,10000 do
+                    local ok, err = dict:safe_set(string.rep("2", 2)..i, string.rep("2", 5)..i)
+                    if not ok then
+                        return
+                    end
+                end
+            end
+            full_dict(cache)
+
+            
+            local lock1 = lock:new("cache_locks", {safe_add = true, timeout = 0})
+            local elapsed, err = lock1:lock("foo")
+            ngx.say("lock: ", elapsed, ", ", err)
+
+            local lock2 = lock:new("cache_locks", {timeout = 0})
+            local elapsed, err = lock2:lock("foo")
+            ngx.say("lock: ", elapsed, ", ", err)
+        ';
+    }
+--- request
+GET /t
+--- response_body
+lock: nil, no memory
+lock: 0, nil
+
+--- no_error_log
+[error]
+
+
+
```

Comment:
There is 3 blank lines between different test cases. Do i need to remove the line #503 ?